packages/api/portfolioDrafts/funding/createFundingStep.test.ts

```diff
@@ -64,126 +54,135 @@ it("sanity check; relative dates used for tests should make sense", () => {
   expect(new Date(today).valueOf() + millisInDay).toEqual(new Date(tomorrow).valueOf());
 });
 
-describe("Handle service level error", () => {
+describe("Handle service level error", function () {
```

Comment:
A comment to open a discussion to help with consistency with tests. After looking at some of the changes from PR #375 I noticed we are going with `() => {}` instead of using `function () {}`. @jasonburkert-ccpo are we moving towards `() => {}` across the codebase? I saw `function () {}` previously in a test and used it because it helped me differentiate the `describe` blocks from the `it` blocks. I just want to make sure we are on the same page so we don't go back and forth between the two.

Reply:
Yes -- we should be consistent about using `() => {}` for `describe` and `async () => {}` for `it`

Reply:
Agree with Kyle above.  Tried to standardize the describes in last PR.

Reply:
Okay, sounds good. I'll make sure to use `() => {}` moving forward and make changes on my branch as well.